content/docs/addons-test-utils.md

```diff
@@ -113,7 +113,7 @@ it('can render and update a counter', () => {
   expect(label.textContent).toBe('You clicked 0 times');
   expect(document.title).toBe('You clicked 0 times');
 
-  // Test second render and componentDidUpdate
+  // 測試第二個 render 和 componentDidUpdate
```

Comment:
Same as above

Reply:
Fix in commit https://github.com/reactjs/zh-hant.reactjs.org/pull/56/commits/ce787f84ead7da09cd235624a12c6c32cdb26390.